keyboards/thevankeyboards/roadkit/keymaps/wavebeem-gamepad/keymap.c

```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_4x4(
+    KC_TAB,  KC_Q,    KC_W,    KC_E,
+    KC_ESC,  KC_A,    KC_S,    KC_D,
+    KC_LSFT, KC_Z,    KC_X,    KC_C,
+    KC_F,    KC_LALT, KC_LCTL, KC_SPC
+  ),
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer